packages/legend-studio/src/components/editor/edit-panel/mapping-editor/MappingTestEditor.tsx

```diff
@@ -333,6 +356,57 @@ export const MappingTestRelationalInputDataBuilder = observer(
   },
 );
 
+export const MappingTestInputDataTypeBuilder = observer(
+  (props: {
+    inputDataState:
+      | MappingTestObjectInputDataState
+      | MappingTestRelationalInputDataState;
+    isReadOnly: boolean;
+  }) => {
+    const { inputDataState, isReadOnly } = props;
+    const inputTypeList: string[] = [];
```

Comment:
Can we just make inputTypeList a list of `ObjectInputType`

Reply:
We cannot do so since we handle two kinds of enum, but I have tried to simplify the logic a bit